en/ui-system/components/editor/graphics/lineJoin.md

```diff
@@ -0,0 +1,26 @@
+# Line Join
+
+The `lineJoin()` method sets or returns the style of the line end cap.
```

Comment:
The `lineJoin` property represents the style of the joint between two line segments.